crates/red_knot_python_semantic/src/types/type_ordering.rs

```diff
@@ -26,6 +26,11 @@ pub(super) fn union_elements_ordering<'db>(left: &Type<'db>, right: &Type<'db>)
     }
 
     match (left, right) {
+        (Type::Callable(CallableType::General(_)), _)
+        | (_, Type::Callable(CallableType::General(_))) => {
+            todo!()
+        }
+
```

Comment:
Any reason not to put this down next to the other `Type::Callable` entries?

Reply:
No reason, I missed this while removing the `todo!()` for the new variant. On that note, I've implemented this to follow the same ordering as other callable types.